src/stories/Library/opening-hours-sidebar/opening-hours-sidebar.scss

```diff
@@ -0,0 +1,70 @@
+@mixin opening-hours-sidebar-padding {
+  padding: $s-lg $s-md;
+
+  @include media-query__small {
+    padding: $s-lg $s-xl;
+  }
+}
+
+.opening-hours-sidebar__header {
+  @include opening-hours-sidebar-padding;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.opening-hours-sidebar__texts {
+  align-items: end;
+  text-align: end;
+}
+
+.opening-hours-sidebar__title {
+  @include typography($typo__h4);
+}
+
+.opening-hours-sidebar-details {
+  @include opening-hours-sidebar-padding;
+  border-top: 1px solid $color__global-tertiary-1;
+}
+
+.opening-hours-sidebar-summary {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.opening-hours-sidebar-summary__icon {
+  transition: transform 0.3s;
+}
+
+.opening-hours-sidebar-summary[aria-expanded="true"]
+  .opening-hours-sidebar-summary__icon {
+  transform: rotateZ(180deg);
+}
```

Comment:
I wonder why this is necessary. I would have expected it to be handled by our disclosure component.

Reply:
There is generally some confusion in our CSS regarding these arrows. I suggest we create a separate ticket to standardize the logic in both CMS and React.

Reply:
https://reload.atlassian.net/browse/DDFHER-75